jetcd-resolver-dns-srv/src/main/java/com/coreos/jetcd/resolver/dnssrv/DnsSrvUriResolver.java

```diff
@@ -50,7 +48,7 @@
     ENV.put("java.naming.provider.url", "dns:");
   }
 
-  private ConcurrentMap<String, EquivalentAddressGroup> cache;
+  private ConcurrentMap<String, SocketAddress> cache;
```

Comment:
why changing `EquivalentAddressGroup` to `SocketAddress`?

Reply:
To reduce packages inter dependencies in OSGi

Reply:
got it.